Libraries/react-native/react-native.js

```diff
@@ -10,14 +10,18 @@
  */
 'use strict';
 
+var React = require('React');
+
 // Export React, plus some native additions.
 //
 // The use of Object.create/assign is to work around a Flow bug (#6560135).
 // Once that is fixed, change this back to
 //
-//   var ReactNative = {...require('React'), /* additions */}
+//   var ReactNative = {...React, /* additions */}
 //
-var ReactNative = Object.assign(Object.create(require('React')), {
+var ReactNative = Object.assign(Object.create(React), {
+  React,
```

Comment:
@vjeux I think so too. Perhaps the question we should answer is whether destructuring assignment is the right pattern for the react-native module.

In web React's case, we usually write `var React = require('react')` and JSX handles the the basic DOM components, so we don't write `var {div, span} = require('react');`.

With React Native, does it make more sense to lowercase the basic components like View, Text, Image, and maybe Touchable* and let JSX handle them? Other components and libraries like Navigator and Vibration would be published as separate packages, which helps reduce the size of the JS bundle as well. The code would look like:
```js
var React = require('react-native');
var Navigator = require('react-native-navigator');
var Vibration = require('react-native-vibration');
...
render() {
  return <view style={styles.container}><text>hi</text></view>;
}
...
var styles = React.createStyleSheet({
  container: {flex: 1}
});